libfs/fs.go

```diff
@@ -459,6 +515,13 @@ func (fs *FS) ensureParentDir(filename string) error {
 	return nil
 }
 
+func (fs *FS) checkEmpty() error {
```

Comment:
Unless there's a convention I'm unaware of, I feel like this would be more readable if it were called `requireNonempty` or something.